08_cell_line_prediction/plot_ridge_results.ipynb

```diff
@@ -0,0 +1,1215 @@
+{
```

Comment:
<p>Is there a reason pancan f-test goes down as the number of features increases? Is it because it has a harder time selecting the best ones?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Some of it is probably noise (like how pancan f-test goes up and down in the second and third plots) but it is interesting that the best performers overall are the f-test methods with fewer features. This dataset is quite a bit smaller than TCGA (in terms of the number of samples, i.e. cell lines) so my guess is that all the methods probably just start overfitting, relative to the more aggressively feature selected models, when more features are included.</p><p> </p><p>I'd like to see if the same thing happens with the TCGA data; my guess is the inflection point will be higher (more features included) since there are more samples.</p>